types/jquery/index.d.ts

```diff
@@ -2389,12 +2390,24 @@ interface JQuery<TElement extends Node = HTMLElement> {
      * @since 1.4
      */
     wrapInner(wrappingElement: JQuery.Selector | JQuery.htmlString | Element | JQuery | ((this: TElement, index: number) => string | JQuery | Element)): this;
+
+    [n: number]: TElement;
 }
 
-interface JQuery<TElement extends Node = HTMLElement> extends ArrayLike<TElement>, Iterable<TElement> { }
+interface JQuery<TElement extends Node = HTMLElement> extends Iterable<TElement> { }
```

Comment:
Interesting, it really does have a proper `Symbol.iterator`.

It is also `ArrayLike`, right?